client/src/modules/common/LabeledInput.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Input from './Input'
+import Label from './Label'
+
+// import "./LabeledInput.css"
+
+class LabeledInput extends React.Component {
+    render() {
+        return (
+            <div className={"labeled-input"}>
+                <Label
+                    displayName = {this.props.displayName}
+                />
+                <Input
+                    value={this.props.value}
+                    placeholder={this.props.placeholder}
+                    onChange={this.props.onChange}
+                />
+                <br/>
```

Comment:
I think this is not correct to put br tag in component

Reply:
Should i use CSS and '\n'
{white-space: pre-line;}

Reply:
Because i didn't implemented any styling. Can I just remove "br" tag , and in future will use css for desired look-and-feel.